clarity/contracts/vaults-v2/arkadiko-vaults-tokens-v1-1.clar

```diff
@@ -0,0 +1,191 @@
+;; Vaults Collateral Tokens 
+;;
+
+;; ---------------------------------------------------------
+;; Constants
+;; ---------------------------------------------------------
+
+(define-constant ERR_NOT_AUTHORIZED u970401)
+(define-constant ERR_UNKNOWN_TOKEN u970001)
+
+;; ---------------------------------------------------------
+;; Variables
+;; ---------------------------------------------------------
+
+(define-data-var token-list (list 25 principal) (list))
```

Comment:
Q: why do we need both a token list and a token map? Lists are slower than maps and from what I see we can just check if the `tokens` map has been set?

Reply:
A: The list is used in `arkadiko-vaults-pool-liquidation-v1-1`. It's used to iterate over all collateral types and claim liquidation rewards for each collateral token.